features/v1_features/latest_version_features/getting_a_job.feature

```diff
@@ -20,6 +20,17 @@ Feature: Getting a job
       | total_search_documents          | 0                         |
       | total_inserted_search_documents | 0                         |
 
+  Scenario: When request is made with empty job id and request returns StatusNotFound by gorilla/mux as handler is not found
```

Comment:
(minor): I guess you probably don't need to specify the router library (gorilla/mux) here as this component test should run against any router library and should not be dependent on a specific library.

Reply:
Done